samples/attested_tls/common/mbedtls_utility.cpp

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Open Enclave SDK contributors.
```

Comment:
Is there a reason this is needs to be a .cpp file instead of a .c file?
C samples can be used in either C or C++

Reply:
I need to check this further. Changing mbedtls_utility.cpp file name to mbedtls_utility.c  is giving linker error that symbols mentioned in the file were not found. I made necessary change .cpp to .c in CMakeList file too.